### ISSUE TITLE
Help To Connect Condenser On Private Customized Steem Blockchain

### ISSUE THREAD

**Issue by rabbykst** [steemit/condenser#3439]
Please give me instruction that how can i connect condenser on my private customized steem blockchain. I need how to configure and connect condenser api. how to add RPC node how to connect condenser wallet on private steem blockchain

**Comment by danirabbani90** [comment]
Seems like no one is even bothered to help about the  private blockchain condenser but i will do my part i wont be rude and harsh like them and will document everything for the newcomers to setup their own private chain

**Comment by KINGdotNET** [comment]
I created a procedure using condenser. 
https://que.com/howto-step-by-step-guide-to-build-your-own-condenser-like-steeming-com-social-network/

I hope that helps.

**Comment by danirabbani90** [comment]
> I created a procedure using condenser.
> https://que.com/howto-step-by-step-guide-to-build-your-own-condenser-like-steeming-com-social-network/
> 
> I hope that helps.

just a quick question ?? can you also please let us know like what we need to add for our private chain ? for example connectSrc”: “‘self’ steemit.com https://api.steemit.com https://api.steem.house api.blocktrades.us https://steemitimages.com&#8221;,
“img_proxy_prefix”: “https://steemitimages.com/&#8221;, // by default it is pointing to https://steemitdevimages.com
“connectSrc”: “‘self’ steemit.com api.steemit.com https://steemitimages.com api.blocktrades.us”, // pre-loaded. Please

**Comment by KINGdotNET** [comment]
For steem, it should be api.steemit.com.

**Comment by danirabbani90** [comment]
> For steem, it should be api.steemit.com.

its a private chain  . so i need to put it the rpc endpoint of it correct ?

**Comment by KINGdotNET** [comment]
If you want it truely private, then you need to create your own full node for your website.

**Comment by danirabbani90** [comment]
yes i Have but i dont have a website . secondly , your step five where it says node https://steeming.org doesnt work ?? can you please let me know how can i generate my session key

**Comment by danirabbani90** [comment]
kingdot net unfortunately your documentation on connecting condenser to private chain is not useful . i am trying to resolve the way for the new condenser and will update the documentation on medium .

**Comment by KINGdotNET** [comment]
Which part is not working? You can leave a comment with tech details and it will be corrected.

**Comment by danirabbani90** [comment]
Kingdotnet when we connect to our private chain it gives error : call bridge get_ranked_posts {"sort":"trending","tag":"","observer":null}
State/store error:  RPCError: Assert Exception:api_itr != _registered_apis.end(): Could not find API bridge RPCError: Assert Exception:api_itr != _registered_apis.end(): Could not find API bridge
 whereas in config.ini rpc plugin is intstalled?

**Comment by tiago-a-ribeiro** [comment]
I have been trying to do the same thing as you @danirabbani90 I have a private steemd full node setup while trying to setup condenser, and I am in a similar situation where condenser just says:
GSA / null true
call bridge get_ranked_posts {"sort":"trending","tag":"","observer":null}

I have the steemd_connection_server and steemd_connection_client set to https://127.0.0.1:9876, the same endpoint I have configured on my node, it's the only address that doesn't just throw a 500 error, but it still doesn't render anything.

Just curious about your current progress on this since a year ago and if you found a solution as I am in the same situation rn.

**Comment by ety001** [comment]
> I have been trying to do the same thing as you @danirabbani90 I have a private steemd full node setup while trying to setup condenser, and I am in a similar situation where condenser just says: GSA / null true call bridge get_ranked_posts {"sort":"trending","tag":"","observer":null}
> 
> I have the steemd_connection_server and steemd_connection_client set to https://127.0.0.1:9876, the same endpoint I have configured on my node, it's the only address that doesn't just throw a 500 error, but it still doesn't render anything.
> 
> Just curious about your current progress on this since a year ago and if you found a solution as I am in the same situation rn.

You need build a Hivemind node for `get_ranked_posts` api. Most of Api which condenser uses are served through Hivemind.
Hivemind: https://github.com/steemit/hivemind
